test/nuts/delete/source.nut.ts

```diff
@@ -265,4 +268,21 @@ describe('project delete source NUTs', () => {
     expect(await isNameObsolete(testkit.username, 'LightningComponentBundle', 'brokerCard')).to.be.false;
     expect(fs.existsSync(brokerPath)).to.be.true;
   });
+
+  (platform() === 'win32' ? it : it.skip)('deletes a remote-only layout using interactive prompt', async () => {
```

Comment:
this one is failing on windows, did you want to actually exclude it from running on linux?
https://github.com/salesforcecli/plugin-deploy-retrieve/actions/runs/8838723739/job/24270580138?pr=988

Reply:
oops, yeah, it's backwards